src/main/resources/application.yaml

```diff
@@ -84,7 +84,7 @@ idam:
     url: ${S2S_URL:http://rpe-service-auth-provider-aat.service.core-compute-aat.internal}
     pbaFromUserAccountMap: false
   s2s-authorised:
-    services: ${PRD_S2S_AUTHORISED_SERVICES:rd_professional_api,rd_user_profile_api,xui_webapp,finrem_payment_service,fpl_case_service,iac,aac_manage_case_assignment,divorce_frontend}
+    services: ${PRD_S2S_AUTHORISED_SERVICES:rd_professional_api,rd_user_profile_api}
```

Comment:
we have moved this to flux , will it override here ?

Reply:
Same change will need to be made in Flux, will create PR now

Reply:
Here is the Cnp-Flux PR: https://github.com/hmcts/cnp-flux-config/pull/7671